src/main/java/com/rackspace/ceres/app/services/DataWriteService.java

```diff
@@ -138,4 +146,13 @@ private void cleanTags(Map<String, String> tags) {
         .retryWhen(appProperties.getRetryInsertDownsampled().build())
         .checkpoint();
   }
+
+  private Mono<?> storeMetricGroup(String tenant, Metric metric) {
+    String updatedAt = metric.getTimestamp().toString();
```

Comment:
What is the use of this updatedAt timestamp?

Reply:
We need to know if this metric group gets updated or not. If not updated for 2 years we can remove it.

Reply:
Why are we using the timestamp from metric data here and not the current timestamp when the metric is being inserted into the table?

Reply:
For easier testing, if we are thinking 2 years I think the accuracy doesn't need to be in millisecs or even seconds?